scripts/select-region-for-e2e-test.ts

```diff
@@ -0,0 +1,40 @@
+import crypto from 'crypto';
+import { AWS_REGIONS_TO_RUN_TESTS } from './cci-utils';
+
+/**
+ * This script prints region assignment for an e2e test.
+ * The algorithm takes two inputs via environment variables - TEST_SUITE and CLI_REGION_SALT
+ * and computes deterministic but random region assignment.
+ * In order to reshuffle regions CLI_REGION_SALT should be modified in workflow yml definition.
+ *
+ * If region is already assigned, i.e. CLI_REGION environment variable is set this script is pass-through.
+ */
+
+const toNumber = (buf: Buffer) => {
+  return buf.readUInt16BE(0) * 0xffffffff + buf.readUInt32BE(2);
+};
+
+// Algorithm from https://github.com/watson/hash-index/blob/fd5d0606926a821166e428b6e266f03b4dc5d817/index.js#L19
```

Comment:
Decided to copy as algorithm is small and library has not been released in years (it doesn't have deps and is simple so no wonder there).